api/net/Areas/Admin/Controllers/OrganizationController.cs

```diff
@@ -0,0 +1,124 @@
+using System.Net;
+using System.Net.Mime;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+using TNO.API.Areas.Admin.Models.Organization;
+using TNO.API.Models;
+using TNO.DAL.Services;
+using TNO.Keycloak;
+
+namespace TNO.API.Areas.Admin.Controllers;
+
+/// <summary>
+/// OrganizationController class, provides organization endpoints for the api.
+/// </summary>
+[ClientRoleAuthorize(ClientRole.Administrator)]
+[ApiController]
+[Area("admin")]
+[ApiVersion("1.0")]
+[Route("api/v{version:apiVersion}/[area]/organizations")]
+[Route("api/[area]/organizations")]
+[Route("v{version:apiVersion}/[area]/organizations")]
+[Route("[area]/organizations")]
+[ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.Unauthorized)]
+[ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.Forbidden)]
+public class OrganizationController : ControllerBase
```

Comment:
We don't need this yet, but it will provide us a way to group user accounts and ministers.